### PR TITLE
en, zh: add clarify for storageVolumes

### PR DESCRIPTION
**Pull request by july2993**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
add clarify for storageVolumes

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by qiancai** [comment]
/retitle en, zh: add clarify for storageVolumes
/translation done
/label needs-cherry-pick-1.1
/cc @DanielZhangQD

File: zh/configure-a-tidb-cluster.md [hunk 1]
```diff
@@ -87,6 +87,10 @@ TiDB Operator 支持为 PD、TiDB、TiKV 挂载多块 PV，可以用于不同用
 
 每个组件都可以配置 `storageVolumes` 字段，用于描述用户自定义的多个 PV。
 
+> **注意：**
+>
+> `storageVolumes` 只支持创建集群的时间配置，后续不支持修改。
```
Comment (DanielZhangQD) [inline zh/configure-a-tidb-cluster.md:92]:
```suggestion
> 请在集群创建之前配置好 `storageVolumes`，集群创建完成后，不支持增加或者删除 `storageVolumes`，对于已经配置的 `storageVolumes`，除增大 `storageVolume.storageSize` 外，其他项不支持修改。如果要增大 `storageVolume.storageSize`，需要对应的 StorageClass 支持[动态扩容](https://kubernetes.io/blog/2018/07/12/resizing-persistent-volumes-using-kubernetes/)。
```

File: en/configure-a-tidb-cluster.md [hunk 2]
```diff
@@ -541,6 +541,10 @@ Configure the `TidbCluster` CR as the following example. In the example, TiDB Op
 
 For how to configure the `spec.tidb.storageVolumes` field, refer to [Multiple disks mounting](#multiple-disks-mounting).
 
+> **Warning:
+>
+> It's only support to configure `storageVolumes` when you create the cluster, do not support update it later once cluster created.
```
Comment (DanielZhangQD) [inline en/configure-a-tidb-cluster.md:546]: @qiancai PTAL the English version, LGTM for the Chinese version.

Comment (qiancai) [inline en/configure-a-tidb-cluster.md:546]:
```suggestion
> You need to configure `storageVolumes` before creating the cluster. After the cluster is created, adding or removing `storageVolumes` is no longer supported. For the `storageVolumes` already configured, except for increasing `storageVolume.storageSize`, other modifications are not supported. To increase `storageVolume.storageSize`, you need to make sure that the corresponding StorageClass supports [dynamic scaling](https://kubernetes.io/blog/2018/07/12/resizing-persistent-volumes- using-kubernetes/).
```

File: en/configure-a-tidb-cluster.md [hunk 3]
```diff
@@ -541,6 +541,10 @@ Configure the `TidbCluster` CR as the following example. In the example, TiDB Op
 
 For how to configure the `spec.tidb.storageVolumes` field, refer to [Multiple disks mounting](#multiple-disks-mounting).
 
+> **Warning:
+>
+> You need to configure `storageVolumes` before creating the cluster. After the cluster is created, adding or removing `storageVolumes` is no longer supported. For the `storageVolumes` already configured, except for increasing `storageVolume.storageSize`, other modifications are not supported. To increase `storageVolume.storageSize`, you need to make sure that the corresponding StorageClass supports [dynamic scaling](https://kubernetes.io/blog/2018/07/12/resizing-persistent-volumes- using-kubernetes/).
```
Comment (july2993) [inline en/configure-a-tidb-cluster.md:546]:
```suggestion
> You need to configure `storageVolumes` before creating the cluster. After the cluster is created, adding or removing `storageVolumes` is no longer supported. For the `storageVolumes` already configured, except for increasing `storageVolume.storageSize`, other modifications are not supported. To increase `storageVolume.storageSize`, you need to make sure that the corresponding StorageClass supports [dynamic scaling](https://kubernetes.io/blog/2018/07/12/resizing-persistent-volumes-using-kubernetes/).
```

File: en/configure-a-tidb-cluster.md [hunk 4]
```diff
@@ -541,6 +541,10 @@ Configure the `TidbCluster` CR as the following example. In the example, TiDB Op
 
 For how to configure the `spec.tidb.storageVolumes` field, refer to [Multiple disks mounting](#multiple-disks-mounting).
 
+> **Warning:
+>
+> You need to configure `storageVolumes` before creating the cluster. After the cluster is created, adding or removing `storageVolumes` is no longer supported. For the `storageVolumes` already configured, except for increasing `storageVolume.storageSize`, other modifications are not supported. To increase `storageVolume.storageSize`, you need to make sure that the corresponding StorageClass supports [dynamic scaling](https://kubernetes.io/blog/2018/07/12/resizing-persistent-volumes-using-kubernetes/).
```
Comment (DanielZhangQD) [inline en/configure-a-tidb-cluster.md:546]:
```suggestion
> You need to configure `storageVolumes` before creating the cluster. After the cluster is created, adding or removing `storageVolumes` is no longer supported. For the `storageVolumes` already configured, except for increasing `storageVolume.storageSize`, other modifications are not supported. To increase `storageVolume.storageSize`, you need to make sure that the corresponding StorageClass supports [dynamic expansion](https://kubernetes.io/blog/2018/07/12/resizing-persistent-volumes-using-kubernetes/).
```

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**